.idea/gradle.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />

```

Comment:
Is this change required for the purpose of this patch or is it some local change. If not, can we remove it from this patch.

Reply:
Yeah, will check if we can remove, if it isn't a lot of trouble. anyways, this pr will be gone, i will merge this change with your branch, and the result will be merged with the main branch. Anyways this is partial success. Will have to work on the server related logic now